sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalog.scala

```diff
@@ -128,6 +128,8 @@ trait ExternalCatalog {
 
   def getTable(db: String, table: String): CatalogTable
 
+  def getTablesByName(db: String, tables: Seq[String]): Seq[CatalogTable]
```

Comment:
When adding a new API like this, we need to add various test cases in ExternalCatalogSuite that is extended by HiveExternalCatalogSuite and InMemoryCatalogSuite.

Reply:
fixed